RELEASE.md

```diff
@@ -80,7 +80,10 @@ Maintaining the release branches for older minor releases happens on a best effo
 
 A few days before a major or minor release, consider updating the dependencies.
 
-Then create a pull request against the main branch.
+Note that we use [Dependabot](.github/dependabot.yml) continuously update most things automatically. Therefore, most dependencies should be up to date already.
```

Comment:
A "to" is missing here somewhere. How about "Note that we use [Dependabot](.github/dependabot.yml) to continuously update…"?